modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -155,3 +156,31 @@ VM_NAME='fcos-node01'
 govc device.serial.add -vm "${VM_NAME}"
 govc device.serial.connect -vm "${VM_NAME}" "[datastore] ${VM_NAME}/console.log"
 ----
+
+== Modifying OVA metadata
+
+NOTE: While we provide these instructions for modifying the OVA, we cannot
+guarantee that any modifications to the OVA metadata will result in a usable
+guest VM.
+
+Fedora CoreOS ships the VMware OVA with certain metadata defaults in the OVA file.
+The hardware version and default firmware are of particular interest, as they
+can affect how the guest VM operates in the hypervisor. These defaults may
+prevent the use of FCOS as a guest VM in older versions of VMware products and
```

Comment:
Should we explicitly state a policy that we will support non-EOL versions of VMware products?

Reply:
I think that is fair; though we don't have any regular testing on those platforms.

Reply:
Is this something we should discuss more broadly before that kind of statement?

Reply:
> I think that is fair; though we don't have any regular testing on those platforms.

I meant in the limited sense that we won't bump the HW version to exclude non-EOL products.  While we're telling users how to change the defaults, it may be helpful to mention when they won't need to.

> Is this something we should discuss more broadly before making that kind of statement?

Probably.